.travis/post_before_script.sh

```diff
@@ -16,7 +16,7 @@ mongo pulp_database --eval 'db.createUser({user:"travis",pwd:"travis",roles:["re
 
 # Otherwise, pulpcore will get installed as the stable release from PyPI
 # as a dep of pulp-2to3-migration.
-pip install $TRAVIS_BUILD_DIR/../pulpcore
+#pip install $TRAVIS_BUILD_DIR/../pulpcore
 
-pip install .
+#pip install .
```

Comment:
@dkliban , I'm trying to figure out why it needs to be installed outside of  the container, any hints?

Reply:
It seems that we need configs for functional tests  from both pulpcore settings and pulp-2to3-migration settings